org.eclipse.lsp4e/build.properties

```diff
@@ -10,3 +10,9 @@ bin.includes = META-INF/,\
                schema/,\
                OSGI-INF/
 src.includes = schema/
+
+# https://codeiseasy.wordpress.com/2013/03/08/tycho-and-jdt-null-analysis/
+# JDT Null Analysis for Eclipse
+additional.bundles = org.eclipse.jdt.annotation
```

Comment:
the blogpost does not say why this preferable to having it in the MANIFEST with optional resolution. Do you know the reason?

Reply:
My understanding is that MANIFEST entries with optional resolution are still evaluated during runtime but we really don't need that bundle during runtime. Dependencies in build.properties are strictly used during compilation/build.